mica/stats/update_guide_stats.py

```diff
@@ -407,8 +407,8 @@ def _get_obsids_to_update(check_missing=False, table_file=None, start=None, stop
             h5 = tables.open_file(table_file, "r")
             tbl = h5.root.data[:]
             h5.close()
-        except:
-            raise ValueError
+        except Exception as err:
+            raise ValueError from err
```

Comment:
Did you consider raise from None? This is also another case where the exception is changed for no apparent reason.